src/dashboard/apigateway/apigateway/controller/crds/v1beta1/convertors/resource.py

```diff
@@ -103,6 +110,56 @@ def _convert_http_resource(self, resource: Dict[str, Any]) -> Optional[BkGateway
             ),
         )
 
+    def _get_version_route_resource(self) -> dict:
```

Comment:
_get_version_route_resource to _get_release_version_route_resource

Reply:
done